parsons/salesforce/salesforce.py

```diff
@@ -12,33 +12,72 @@ class Salesforce:
     """
     Instantiate the Salesforce class
 
+    Supports the password and client_credentials authentication_methods
```

Comment:
I think the underscore between "authentication" and "methods" is a typo?

Is there public-facing documentation about how to get `client_credentials` auth info? Like do you have to request a key? If so we should update the docs page for the salesforce connector and/or put a link to that page here.

Reply:
thanks! fixed type and added link!